ledger/participant-integration-api/src/main/scala/platform/store/interning/StringInterningView.scala

```diff
@@ -28,9 +28,15 @@ trait UpdatingStringInterningView {
   /** Update the StringInterningView from persistence
     *
     * @param lastStringInterningId this is the "version" of the persistent view, which from the StringInterningView can see if it is behind
-    * @return a completion Future: if the view is behind it will load the missing entries from persistence, and update the view state
+    * @return a completion Future:
+    *
+    * * if the view is behind, it will load the missing entries from persistence, and update the view state.
+    *
+    * * if the view is ahead, it will remove all entries with ids greater than the `lastStringInterningId`
     */
-  def update(lastStringInterningId: Int)(implicit loggingContext: LoggingContext): Future[Unit]
+  def update(lastStringInterningId: Int)(
```

Comment:
can you please add some comments the update is not thread-safe: not for concurrent usage, and that the internize and read functionality is thread safe.

Reply:
good point. Thanks for the reminder: added